src/v0/destinations/custify/deleteUsers.js

```diff
@@ -17,36 +17,40 @@ const userDeletionHandler = async (userAttributes, config) => {
   }
 
   const { apiKey } = config;
-  const { userId } = userAttributes;
 
   if (!apiKey) {
     throw new ConfigurationError('api key for deletion not present', 400);
   }
-  if (!userId) {
-    throw new InstrumentationError('User id for deletion not present', 400);
-  }
-  const requestUrl = `https://api.custify.com/people?user_id=${userId}`;
-  const requestOptions = {
-    headers: {
-      Authorization: `Bearer ${apiKey}`,
-    },
-  };
+  await Promise.all(
+    userAttributes.map(async (userAttr) => {
+      const { userId } = userAttr;
+      if (!userId) {
+        throw new InstrumentationError('User id for deletion not present', 400);
+      }
+      const requestUrl = `https://api.custify.com/people?user_id=${userId}`;
```

Comment:
Can you add some ref destination docs in comment here

Reply:
Done